google/cloud/container/quickstart/quickstart.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/container/cluster_manager_client.h"
+#include "google/cloud/project.h"
+#include <iostream>
+#include <stdexcept>
+
+int main(int argc, char* argv[]) try {
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " project-id\n";
+    return 1;
+  }
+
+  namespace container = ::google::cloud::container;
+  auto client = container::ClusterManagerClient(
+      container::MakeClusterManagerConnection(/* EDIT HERE */));
+
+  auto const project = google::cloud::Project(argv[1]);
```

Comment:
looks like we want `project + "/locations/-"`

https://github.com/googleapis/google-cloud-cpp/blob/8a68ee4c1d8d16d81496dd25fb2e47b20cbf6b72/google/cloud/container/cluster_manager_client.h#L93-L96